install/0000_30_machine-api-operator_09_rbac.yaml

```diff
@@ -332,6 +332,21 @@ subjects:
     name: prometheus-k8s
     namespace: openshift-monitoring
 
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: machine-api-controllers
```

Comment:
There is already a RoleBinding with this name.  The name here should be cloud-provider-config-reader I think, or possibly something like 'machine-api-cloud-provider-config-reader' so there aren't collisions with other components.

Reply:
There was a prior suggestion to have "machine-api" in the name to show ownership, so I will go with "machine-api-cloud-provider-config-reader". Thanks!

Reply:
I'd rather keep the name as `machine-api-controllers` so we are consistent across  with the existing ones and we can clearly identify owned by this component in each namespace, not a blocker...